core_lib/parking.py

```diff
@@ -1,9 +1,162 @@
 import cv2
+import json
 import numpy as np
+import math
+import statistics
+
 
 from os import path
 from core_lib.core import get_config_value
+from core_lib.core import read_training_params
+from core_lib.core import save_config_value
+from core_lib.core import Direction
+from core_lib.video_feed import VideoFeed
+from core_lib.seeds import get_seeds
+from core_lib.segmentation import region_expander
+from core_lib.drawing import draw_region_characteristics
+from core_lib.region_identifier import identify_region
+
+
+def get_final_mapped_points(x, y):
+    """
+    Gets final coordinates given a list of free parking spices and real coordinates
+
+    Returns:
+    parking dict --- dict with the scaled data of a parking spot
+    """
+    target = None
+    free_parking_spaces = get_config_value("free_parking_spaces")
+
+    for space in free_parking_spaces:
+        corner_1 = space["corner_1"]
+        corner_2 = space["corner_2"]
+        if (x >= corner_1[0] and x <= corner_2[0]) and (y >= corner_1[1] and y <= corner_2[1]):
+            target = space
+            x_scale, y_scale = get_scaling_factor()
+
+            for item in target:
+                target[item][0] = round(target[item][0] * x_scale)
+                target[item][1] = round(target[item][1] * y_scale)
+
+            return target
```

Comment:
Solo se necesita el centro para el algoritmo.

```suggestion
            x = int(round(space["algorithm_center"][0] * x_scale))
            y = int(round(space["algorithm_center"][1] * y_scale))
            return x, y
```